WcaOnRails/db/seeds/teams.seeds.rb

```diff
@@ -12,3 +12,8 @@
 Team.create(friendly_id: 'wst', email: "software@worldcubeassociation.org")
 Team.create(friendly_id: 'banned', email: "disciplinary@worldcubeassociation.org", hidden: true)
 Team.create(friendly_id: 'wcat', email: "competitions@worldcubeassociation.org")
+Team.create(friendly_id: 'chair')
```

Comment:
I'm not sure if there is any code that assumes that teams have email addresses. I'm ok to leave this as is, but something we should keep in mind in case anything breaks after we deploy this.

Reply:
There's no validation for that on the team model, but I don't know if it could be an issue in some place I haven't looked at.